db/migrate/20170818162059_add_email_confirm_column_to_users.rb

```diff
@@ -0,0 +1,6 @@
+class AddEmailConfirmColumnToUsers < ActiveRecord::Migration[5.0]
+  def change
+    add_column :users, :email_confirmed, :boolean, :default => false
+    add_column :users, :confirm_token, :string
```

Comment:
I looked around and everyone had them on the same table. If I ended up changing it a few functions would break so I'm leaving it as is for the time being.